libs/sdk-backend-spi/src/organization/settings/index.ts

```diff
@@ -26,6 +26,24 @@ export interface IOrganizationSettingsService {
      */
     setLocale(locale: string): Promise<void>;
 
+    /**
+     * Sets timezone for organization.
+     *
+     * @param locale - for example "en-US", "cs-CZ", etc.
+     *
+     * @returns promise
+     */
+    setTimeZone(timezoneId: string): Promise<void>;
```

Comment:
- Should we change the function name `setTimeZone` → `setTimezone`?
  `Timezone` itself is a valid word, there is no need to capitalize the letter `Z`, is there?

- Should we change the param name `timezoneId` → `timezone`?

Reply:
fixed